### PR TITLE
Add broadcast messages

### PR DESCRIPTION
**Pull request by harish-aka-shivi**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/Modules/broadcastOperations.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+  const db = firebase.firestore();
+
+  if (!originalBroadcast) {
+    throw new Error("Original broadcast message is not defined");
+  };
+
+  const broadcastToBeSaved = {
+    id: broadcastMessageId,
+    owner: userId,
+    creationDate: firebase.firestore.FieldValue.serverTimestamp(),
```
Comment (joaoaguiam) [inline src/Modules/broadcastOperations.js:66]: This is overriding the initial creationDate. it is not critical but not super correct either :P

File: src/Modules/broadcastOperations.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+  const broadcastRef = db
+    .collection("eventSessions")
+    .doc(sessionId.toLowerCase())
+    .collection("broadcasts")
+    .doc(namespace)
+    .collection("broadcastsMessages")
+    .doc(`${broadcastMessageId}`);
+
+  batch.update(broadcastRef, broadcastToBeSaved);
```
Comment (joaoaguiam) [inline src/Modules/broadcastOperations.js:81]: this is the only operation, so not needed to use the batch feature from firestore

File: src/Modules/broadcastOperations.js (hunk 1 above)
Comment (harish-aka-shivi) [reply src/Modules/broadcastOperations.js:66]: Agree, will have a look

File: src/Modules/broadcastOperations.js (hunk 2 above)
Comment (harish-aka-shivi) [reply src/Modules/broadcastOperations.js:81]: Agree, will take care of it. 👍

**Pull request merged**